pennylane/workflow/qnode.py

```diff
@@ -1113,14 +1107,15 @@ def _execution_component(self, args: tuple, kwargs: dict, override_shots) -> qml
         full_transform_program.set_classical_component(self, args, kwargs)
         _prune_dynamic_transform(full_transform_program, inner_transform_program)
 
+        orig_mcm_config = self.execute_kwargs.pop("mcm_config")
         with warnings.catch_warnings():
             # TODO: remove this once the cycle for the arguements have finished, i.e. 0.39.
             warnings.filterwarnings(
                 action="ignore",
                 message=r".*argument is deprecated and will be removed in version 0.39.*",
                 category=qml.PennyLaneDeprecationWarning,
             )
-            # pylint: disable=unexpected-keyword-arg
+            # pylint: disable=unexpected-keyword-arg, repeated-keyword

```

Comment:
Does the linter think `mcm_config` is in `execute_kwargs` even though we just popped it?

Reply:
Yes, it's very annoying, though I've changed how the kwargs are being used now so it should be fine to stop suppressing that warning.